examples/next/getting-started/src/faust.config.js

```diff
@@ -9,7 +9,10 @@ if (!process.env.NEXT_PUBLIC_WORDPRESS_URL) {
 /**
  * @type {import("@faustjs/core").Config}
  */
+const secretKey =
+  process.env.FAUST_SECRET_KEY || process.env.FAUSTWP_SECRET_KEY;
```

Comment:
We may want to move `process.env.FAUST_SECRET_KEY || process.env.FAUSTWP_SECRET_KEY` into it's own function